### PR TITLE
Resolve SDB in flat roster

### PR DESCRIPTION
**Pull request by amendlik**
### What does this PR do?
Add SDB support to the flat roster

### What issues does this PR fix or reference?
None

### Previous Behavior
Flat roster did not resolve sdb:// URLs

### New Behavior
Flat roster resolves sdb:// URLs

### Tests written?
No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

### CODE REVIEW

File: salt/roster/flat.py [hunk 1]
```diff
@@ -43,11 +42,36 @@ def targets(tgt, tgt_type='glob', **kwargs):
                            **kwargs)
     conditioned_raw = {}
     for minion in raw:
-        conditioned_raw[str(minion)] = raw[minion]
+        conditioned_raw[str(minion)] = _resolve_sdb(raw[minion])
```
Comment (gtmanfred) [inline salt/roster/flat.py]: Is there a reason to not use `salt.config.apply_sdb`?

**Review by gtmanfred — CHANGES_REQUESTED** [review]

File: salt/roster/flat.py (hunk 1 above)
Comment (amendlik) [inline salt/roster/flat.py]: I did not know that function existed. It does the exact same thing as `_resolve_sdb`, so I'll use `salt.config.apply_sdb`. Thanks!

**Comment by cachedout** [comment]
@gtmanfred Re-review requested.

**Review by gtmanfred — APPROVED** [review]

**Comment by gtmanfred** [comment]
Looks good to me!

**Pull request merged**